### PR TITLE
Azure storage meta package deprecation

### PR DESCRIPTION
**Pull request by rakshith91**
Fixes https://github.com/Azure/azure-sdk-for-python/issues/10641

### CODE REVIEW

File: sdk/storage/azure-storage/setup.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+-  [azure-storage-blob](https://pypi.org/project/azure-storage-blob) : Blob storage client
+- [azure-storage-file-share](https://pypi.org/project/azure-storage-file-share) : Storage file share client
+- [azure-storage-file-datalake](https://pypi.org/project/azure-storage-file-datalake) : ADLS Gen2 client
+- [azure-storage-queue](https://pypi.org/project/azure-storage-queue): Queue storage client
+"""
+
+if "sdist" in sys.argv:
+    setup(
+        name='azure-storage',
+        version='0.37',
```
Comment (rakshith91) [inline sdk/storage/azure-storage/setup.py:32]:
```suggestion
        version='0.37.0',
```

File: sdk/storage/azure-storage/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+https://aka.ms/azsdk/python/all
+
+Here's a non-exhaustive list of common packages:
+
+-  [azure-storage-blob](https://pypi.org/project/azure-storage-blob) : Blob storage client
+- [azure-storage-file-share](https://pypi.org/project/azure-storage-file-share) : Storage file share client
+- [azure-storage-file-datalake](https://pypi.org/project/azure-storage-file-datalake) : ADLS Gen2 client
+- [azure-storage-queue](https://pypi.org/project/azure-storage-queue): Queue storage client
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-python%2Fazure%2FREADME.png)
```
Comment (lmazuel) [inline sdk/storage/azure-storage/README.md:17]: Link needs update

File: sdk/storage/azure-storage/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+This package was the Microsoft Azure Storage bundle.
+
+Starting with v0.37.0, this package is deprecated. Please install the service specific packages prefixed by `azure-storgae` needed for your application.
+
+The complete list of available packages can be found at:
+https://aka.ms/azsdk/python/all
+
+Here's a non-exhaustive list of common packages:
+
+-  [azure-storage-blob](https://pypi.org/project/azure-storage-blob) : Blob storage client
```
Comment (lmazuel) [inline sdk/storage/azure-storage/README.md:12]: extra-space

File: sdk/storage/azure-storage/sdk_packaging.toml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+[packaging]
+auto_update = false
```
Comment (rakshith91) [inline sdk/storage/azure-storage/sdk_packaging.toml:2]:
```suggestion
auto_update = false

```

File: sdk/storage/azure-storage/setup.py [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+Starting with v0.37.0, the 'azure-storage' meta-package is deprecated and cannot be installed anymore.
+Please install the service specific packages prefixed by `azure` needed for your application.
+
+The complete list of available packages can be found at:
+https://aka.ms/azsdk/python/all
+
+Here's a non-exhaustive list of common packages:
+
+-  [azure-storage-blob](https://pypi.org/project/azure-storage-blob) : Blob storage client
```
Comment (lmazuel) [inline sdk/storage/azure-storage/setup.py:23]: extra space

Comment (rakshith91) [inline sdk/storage/azure-storage/setup.py:23]:
```suggestion
- [azure-storage-blob](https://pypi.org/project/azure-storage-blob) : Blob storage client
```

**Comment by ONFU** [comment]
the omith haved by the nursing of tits glooming kitherver rouge northen haved milk of the tity fuck

**Pull request merged**